bbs_plus/src/threshold/zero_sharing.rs

```diff
@@ -23,7 +23,7 @@ use oblivious_transfer_protocols::ParticipantId;
 pub struct Party<F: PrimeField, const SALT_SIZE: usize> {
     pub id: ParticipantId,
     pub protocol_id: Vec<u8>,
-    pub batch_size: usize,
+    pub batch_size: u64,
```

Comment:
Should be u16 or u32 and `Party::init` should accept `batch_size` type as that as well.